experimental/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ For notes on migrating to 2.x / 0.200.x see [the upgrade guide](doc/upgrade-to-2
 
 ### :house: Internal
 
+- update node sdk to use env var util
```

Comment:
Please see the changelog entry formatting example at https://github.com/open-telemetry/opentelemetry-js/pull/5580/files#diff-c9a82ab13ebd74152ffa3cc91afa636f11f19174034c8bc12990ae2a69c2c507R19